src/status_im/contexts/wallet/collectible/events.cljs

```diff
@@ -166,39 +160,58 @@
              [:dispatch [:wallet/flush-collectibles-fetched]])]})))
 
 (rf/reg-event-fx
- :wallet/get-collectible-details
- (fn [{:keys [db]} [collectible-id aspect-ratio]]
+ :wallet/navigate-to-collectible-details
+ (fn [{:keys [db]}
+      [{{collectible-id :id :as collectible} :collectible
+        aspect-ratio                         :aspect-ratio
+        gradient-color                       :gradient-color}]]
    (let [request-id               0
          collectible-id-converted (cske/transform-keys transforms/->PascalCaseKeyword collectible-id)
          data-type                (collectible-data-types :details)
          request-params           [request-id [collectible-id-converted] data-type]]
-     {:db (assoc-in db [:wallet :last-collectible-aspect-ratio] aspect-ratio)
+     {:db (assoc-in db
+           [:wallet :ui :collectible]
+           {:details        collectible
+            :aspect-ratio   aspect-ratio
+            :gradient-color gradient-color})
```

Comment:
Moved the collectible data to `:wallet :ui :collectible`, instead of `:wallet :last-collectible...`